src/utils/blockedMonitor.ts

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License.
+
+import { AzureFunctionsRpcMessages as rpc } from './../../azure-functions-language-worker-protobuf/src/rpc';
+import LogCategory = rpc.RpcLog.RpcLogCategory;
+import LogLevel = rpc.RpcLog.Level;
+import blockedAt = require('blocked-at');
+
+export function startBlockedMonitor(channel: { log: (log: rpc.IRpcLog) => void }, threshold = 100): NodeJS.Timer {
```

Comment:
Since this will have an impact on performance, can we add a warning at the beginning so that users don't forget to turn it off